client/v2/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	cosmossdk.io/x/tx v0.10.0
 	github.com/cockroachdb/errors v1.11.1
 	github.com/cosmos/cosmos-proto v1.0.0-beta.3
-	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230718211500-1d74652f6021
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20230828195211-3a911572e3e1
```

Comment:
can we revert this? there should be no changes in the SDK

Reply:
No, we need the latest changes of hub, 
https://github.com/cosmos/cosmos-sdk/blob/706a531e5a0b97b1082ad872b5c2226866c3c3cf/client/v2/autocli/builder.go#L40

`AddressCodec` is introduced in later commits of existing one.